packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -185,6 +312,85 @@ export const setupRDSInstanceAndData = async (
   return dbConfig;
 };
 
+/**
+ * Creates a new RDS Aurora serverless V2 cluster with one DB instance using the given input configuration, runs the given queries and returns the details of the created RDS
+ * instance.
+ * @param config Configuration of the database cluster
+ * @param queries Initial queries to be executed
+ * @returns Endpoint address, port and database name of the created RDS cluster.
+ */
+export const setupRDSClusterAndData = async (
+  config: RDSConfig,
+  queries?: string[],
+): Promise<{
+  endpoint: string;
+  port: number;
+  dbName: string;
+  dbInstance: DBInstance;
+  password: string;
+  secretArn: string;
+  managedSecretArn: string;
+}> => {
+  console.log(`Creating RDS ${config.engine} DB cluster with identifier ${config.identifier}`);
+
+  const dbCluster = await createRDSCluster(config);
+
+  const { secretArn } = await storeDbConnectionConfigWithSecretsManager({
+    region: config.region,
+    username: config.username,
+    password: dbCluster.password,
+    secretName: `${config.identifier}-secret`,
+  });
+
+  if (!secretArn) {
+    throw new Error('Failed to store db connection config in secrets manager');
+  }
+
+  const client = new RDSDataClient({ region: config.region });
+
+  // create a new test database with given name
+  const createDBInput: ExecuteStatementCommandInput = {
+    resourceArn: dbCluster.clusterArn,
+    secretArn,
+    sql: `create database ${config.dbname};`,
+    database: dbCluster.dbName,
+  };
+
+  const command = new ExecuteStatementCommand(createDBInput);
+  try {
+    const createDBResponse = await client.send(command);
+    console.log('Create database response: ' + JSON.stringify(createDBResponse));
+  } catch (err) {
+    console.log(err);
+  }
+
+  // create the test tables in the test database
+  queries?.map(async (query) => {
+    try {
+      const createTableInput: ExecuteStatementCommandInput = {
+        resourceArn: dbCluster.clusterArn,
+        secretArn,
+        sql: query,
+        database: config.dbname,
+      };
+      const createTableResponse = await client.send(new ExecuteStatementCommand(createTableInput));
+      console.log('Create table response: ' + JSON.stringify(createTableResponse));
+    } catch (err) {
+      console.log(err);
+    }
+  });
+
+  return {
+    endpoint: dbCluster.endpoint,
+    port: dbCluster.port,
+    dbName: dbCluster.dbName,
+    dbInstance: dbCluster.dbInstance,
+    password: dbCluster.password,
+    secretArn: secretArn,
+    managedSecretArn: dbCluster?.managedSecretArn,
```

Comment:
What is the difference between secretArn and managedSecretArn? How can we make that difference easier to understand just by looking at the names?